.github/dependabot.yml

```diff
@@ -20,6 +20,7 @@ updates:
       - dependency-name: org.freemarker:freemarker
       - dependency-name: org.eclipse.jgit:org.eclipse.jgit
       - dependency-name: org.eclipse.jgit:org.eclipse.jgit.http.server
+      - dependency-name: org.eclipse.jgit:org.eclipse.jgit.ssh.jsch
```

Comment:
Shouldn't we just have `org.eclipse.jgit:*`? I tested wildcard and they are working fine.

Reply:
Sure, done 👍🏻